### PR TITLE
add diagram

### PR DESCRIPTION
**Pull request by zb32**
Add workflow diagram to the README

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/blobtoolkit/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/blobtoolkit _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the unit test suite passes (`nextflow run . -profile test,singularity,sanger --outdir <OUTDIR>`) locally.
- [ ] Ensure the full test suite passes (`nextflow run . -profile test_full,singularity,sanger --outdir <OUTDIR>`) locally.
- [ ] Ensure that the CI tests pass by checking the nextflow tower link at the bottom of the CI tests and check the output on farm5 (the CI test only tells you if the job was successfully submitted to nextflow tower).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by priyanka-surana** [comment]
@zb32 Notes from in-person conversation: 
1. Update the workflow to show changes in the actual pipeline, mostly around input for `window_stats`. 
2. Upload svg along with the png.

File: docs/images/sanger-tol-readmapping_workflow.png
Comment (priyanka-surana) [inline docs/images/sanger-tol-readmapping_workflow.png:1]: It says readmapping workflow. Should this be deleted? There is a blobtoolkit workflow as well.

Comment (zb32) [reply docs/images/sanger-tol-readmapping_workflow.png:1]: It's the previous diagram (without `fastawindow` used as a input to `window_stats`) I've deleted it locally but need to delete it here as well

**Review by priyanka-surana — APPROVED** [review]
I am going to merge this into `dev`. The steps around `coverage_tsv` need to be updated, but will do so after all code changes have been merged and tested.

**Pull request merged**